### PR TITLE
"com.android.library" Gradle Module Support

### PR DESCRIPTION
**Pull request by ErickSumargo**
Based on this [issue](https://github.com/google/secrets-gradle-plugin/issues/14).

### CODE REVIEW

File: plugin/src/main/java/com/google/secrets_gradle_plugin/Extensions.kt [hunk 1]
```diff
@@ -15,14 +15,18 @@
...
 import org.gradle.api.Project
 import java.io.FileNotFoundException
 import java.util.Properties
 
 fun Project.androidProject() : AppExtension? =
     extensions.findByType(AppExtension::class.java)
 
+fun Project.libraryProject() : LibraryExtension? =
+    extensions.findByType(LibraryExtension::class.java)
+
```
Comment (ErickSumargo) [inline plugin/src/main/java/com/google/secrets_gradle_plugin/Extensions.kt:29]: There are 2 other plugin types left based on `com.android.build.gradle.BaseExtension` doc:
- `TestExtension`,
- `DynamicFeatureExtension`

But guess I will just stick with my current use case only.

**Comment by ErickSumargo** [comment]
Also, should I extend `:sample-app` with `com.android.library` module case sample?

**Review by arriolac — APPROVED** [review]
This is great! Thanks for the contribution.

**Comment by arriolac** [comment]
> Also, should I extend `:sample-app` with `com.android.library` module case sample?

That shouldn't be necessary, but, can you add a note on the README about this use case?

**Comment by ErickSumargo** [comment]
Done.

**Pull request merged**